### PR TITLE
feat: Admin Provide Option "Not to show on Frontpage" 

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/4963

server side https://github.com/fossasia/open-event-server/pull/7588
Screenshot from 2021-01-09 15-49-54

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/ui-table/cell/admin/events/event-no-front.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import classic from 'ember-classic-decorator';
+import Component from '@ember/component';
+
+@classic
+export default class EventNoFront extends Component {}
```
Comment (iamareebjamal) [inline app/components/ui-table/cell/admin/events/event-no-front.js:5]: As we discussed previously, no new component will be made classic, classic has been deprecated

Comment (maze-runnar) [reply app/components/ui-table/cell/admin/events/event-no-front.js:5]:
```
import Component from '@glimmer/component';

export default class EventNoFront extends Component {}

```
Screenshot from 2021-01-09 20-14-44

Comment (maze-runnar) [reply app/components/ui-table/cell/admin/events/event-no-front.js:5]: why?

Comment (iamareebjamal) [reply app/components/ui-table/cell/admin/events/event-no-front.js:5]: Write the component using new syntax. Remove the JS file, it is useless

Comment (maze-runnar) [reply app/components/ui-table/cell/admin/events/event-no-front.js]: ok

Comment (maze-runnar) [reply app/components/ui-table/cell/admin/events/event-no-front.js]: What is meaning by new syntax? I am using `<UiCheckbox @class="toggle" @checked={{this.extraRecords.isDemoted}} @onChange={{action this.props.actions.toggleDemoted this.record}} />` and it is giving error without js file

File: app/components/ui-table/cell/admin/events/event-no-front.js [hunk 2]
```diff
@@ -1,5 +1,3 @@
-import classic from 'ember-classic-decorator';
 import Component from '@ember/component';
 
-@classic
-export default class EventNoFront extends Component {}
+export default Component.extend({});
```
Comment (iamareebjamal) [inline app/components/ui-table/cell/admin/events/event-no-front.js:3]: Delete this file

File: app/components/ui-table/cell/admin/events/event-no-front.js (hunk 1 above)
Comment (iamareebjamal) [reply app/components/ui-table/cell/admin/events/event-no-front.js]: Octane syntax

**Comment by mariobehling** [comment]
Thank you! Events that are listed in the section "Call for Speakers" seem to show up still even though the option is activated. Therefore reopening the issue.

**Pull request merged**